openupgrade_scripts/scripts/hr_attendance/16.0.2.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Models in module 'hr_attendance'---
+---Fields in module 'hr_attendance'---
+hr_attendance / res.company              / attendance_barcode_source (selection): NEW selection_keys: ['back', 'front', 'scanner'], hasdefault: default
```

Comment:
Please document always why they are nothing to do, and investigate further if the default is adequate or not:

```suggestion
hr_attendance / res.company              / attendance_barcode_source (selection): NEW selection_keys: ['back', 'front', 'scanner'], hasdefault: default
# NOTHING TO DO: Value only used in enterprise

```